admin-ui/plugins/services/Components/Configuration/LdapListPage.tsx

```
@@ -69,122 +104,159 @@ function LdapListPage() {
   )
 
-  const handleLdapDelete = useCallback((row) => {
-    setItem(row)
-    toggle()
-  }, [])
+  const handleLdapDelete = useCallback(
+    (row: GluuLdapConfiguration) => {
+      setItem(row)
+      toggle()
+    },
+    [toggle],
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)